pkg/thanos/operator.go

```diff
@@ -613,7 +616,7 @@ func (o *Operator) UpdateStatus(ctx context.Context, key string) error {
 		return err
 	}
 
-	if tr == nil || o.rr.DeletionInProgress(tr) {
+	if o.rr.DeletionInProgress(tr) {
```

Comment:
we don't need to modify this.
```suggestion
	if tr == nil || o.rr.DeletionInProgress(tr) {
```